### PR TITLE
feat: new repo name

### PR DESCRIPTION
**Pull request by noel-yoo**
## Description
This PR handles the new repo name `dev-wallet`

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] Add tests to cover changes as needed.
- [x] Update documentation as needed.

### CODE REVIEW

File: scripts/ci_deploy.sh [hunk 1]
```diff
@@ -10,7 +10,7 @@ commit=$(git rev-parse --short=7 $TRAVIS_COMMIT)
 
 accountID=$(aws sts get-caller-identity --output text --query 'Account')
 regionID=us-west-2
-application=nucleus-wallet
+application=dev-wallet
```
Comment (joel-zilliqa) [inline scripts/ci_deploy.sh:13]: You will break the entire ci-cd pipeline with this change

**Review by joel-zilliqa — CHANGES_REQUESTED** [review]

File: scripts/ci_deploy.sh (hunk 1 above)
Comment (noel-yoo) [reply scripts/ci_deploy.sh:13]: OK, let's update it when we have updated the CICD.

Comment (noel-yoo) [reply scripts/ci_deploy.sh:13]: As @joel-zilliqa updated the CICD, we can replace it.

Comment (joel-zilliqa) [reply scripts/ci_deploy.sh:13]: I've updated the CICD

**Review by joel-zilliqa — APPROVED** [review]

**Pull request merged**